doc/libpmemobj.3

```diff
@@ -2076,7 +2077,30 @@ in the object specified by
 .I oid
 and saves it to the undo log.  The application is then free to directly modify
 the object in that memory range.  In case of a failure or abort, all the changes
-within this range will be rolled-back.
+within this range will be rolled-back.  The supplied block of memory has to be within
+the given pool.
+If successful and called during
+.I TX_STAGE_WORK
+function returns zero.  Otherwise, state changes to
+.I TX_STAGE_ONABORT
+and an error number is returned.
+.PP
+.BI "int pmemobj_tx_add_range_direct(void *" ptr ", size_t " size ");
+.IP
+The
+.BR pmemobj_tx_add_range_direct ()
+behaves the same as
+.BR pmemobj_tx_add_range ()
+with the exception that it operates on virtual memory addresses and not
+persistent memory objects.  It takes a "snapshot" of a persistent memory block
+of given
+.IR size ,
+located at the given address in the virtual memory space
+.I ptr
```

Comment:
...
+located at the given address
+.I ptr
+in the virtual memory space
...
in my opinion would be better